src/workspace/.env.development

```diff
@@ -1,6 +1,4 @@
 # Reactive Trader url
 VITE_RT_URL=http://localhost:1917
-# Reactive Analytics url
```

Comment:
based on env === "local" instead .. we should be able to run locally off development or production builds!